internal/cli/iam/globalwhitelist/description.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package globalwhitelist
+
+const (
+	short           = "Manage the access whitelist for Global API Key."
+	createWhitelist = "Create an access whitelist for Global API Key."
+	deleteWhitelist = "Delete a database user from Global API Key."
```

Comment:
```suggestion
	deleteWhitelist = "Delete an access whitelist for Global API Key."
```

Reply:
copy/pasta and wrong also in the original, thanks for catching this